src/algebraic_topology/simplicial_object.lean

```diff
@@ -28,39 +32,39 @@ variables (C : Type u) [category.{v} C]
 /-- The category of simplicial objects valued in a category `C`.
 This is the category of contravariant functors from `simplex_category` to `C`. -/
 @[derive category, nolint has_inhabited_instance]
-def simplicial_object := simplex_categoryᵒᵖ ⥤ C
+def simplicial_object := simplex_category.{v}ᵒᵖ ⥤ C
 
 namespace simplicial_object
 
+-- TODO: find a sensible value for `1000`.
```

Comment:
Still todo before we merge?

Reply:
After the discussion on zulip with @pechersky , I think 1000 is good enough. I removed the comment.